backend/app/legacy/crypto/eth_sign/patron_mode.py

```diff
@@ -9,4 +13,6 @@ def build_patron_mode_msg(user_address: str, toggle: bool) -> str:
 def verify(user_address: str, toggle: bool, signature: str) -> bool:
```

Comment:
this is a legacy module, are you sure we should change the verify function here? is it called anywhere?

Reply:
For me it makes sense :)

It's used in `backend/app/legacy/controllers/user.py` as 
`from app.legacy.crypto.eth_sign import patron_mode as patron_mode_crypto`
which is the used as controller in API for toggle.

I was able to remove the part of signature (file removed below) by using the non-legacy implementation